app/src/main/res/layout/busstop_shelter.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is titled `busstop_shelter` but has references to tactilePavingSidewalkImage. This doesn't seem right!